packages/yoroi-extension/chrome/extension/ergo-connector/api.js

```diff
@@ -262,13 +266,40 @@ export async function connectorSignTx(
   const jsonBoxesToSign = utxosToSign.map(formatUtxoToBox);
   processBoxesForSigmaRust(jsonBoxesToSign);
   const txBoxesToSign = RustModule.SigmaRust.ErgoBoxes.from_boxes_json(jsonBoxesToSign);
+  console.log('data inputs');
+  const dataBoxIds = tx.dataInputs.map(box => box.boxId);
```

Comment:
I think we might have to update the spec to include the full ErgoBox for data inputs too to match the unsigned inputs. We never updated the signing code since we primarily made that change for the UI side to display the data. I'm guessing we want to take out these yoroi-supplied utxos and directly use the data from `UnsignedInput` here too? Likewise for `DataInput` if we do the same there.